### PR TITLE
Remove prop-types from prod build

### PR DESCRIPTION
**Pull request by felixmosh**

### CODE REVIEW

**Comment by maxmarinich** [comment]
Hi, @felixmosh! What're the pros of using this approach?

**Comment by felixmosh** [comment]
This is the approach of every major lib which uses prop-types, prop-types ment to be used on development only.

Currently production builds has it inside the bundle (redundant bytes to download & parse).

File: src/lib/prop-types.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import PropTypes from 'prop-types'
 
-export const propTypes = {
+export const propTypes = process.env.NODE_ENV === 'development' ? {
```
Comment (maxmarinich) [inline src/lib/prop-types.js:3]: We must go from the opposite `process.env.NODE_ENV !== 'production'`

**Comment by maxmarinich** [comment]
Don't think the size reduction is worth the surprising behavior here. Every major project uses custom optimization. And if needed reduce bundle size, for example, it can use a [plugin](https://www.npmjs.com/package/babel-plugin-transform-react-remove-prop-types) as a  complex decision. Please update conditions for env's like i specified in comment.

**Pull request merged**